Sources/Build/BuildManifest/LLBuildManifestBuilder+Swift.swift

```diff
@@ -521,7 +537,7 @@ extension LLBuildManifestBuilder {
 
     private func addModuleWrapCmd(_ target: SwiftTargetBuildDescription) throws {
         // Add commands to perform the module wrapping Swift modules when debugging strategy is `modulewrap`.
-        guard target.defaultBuildParameters.debuggingStrategy == .modulewrap else { return }
+        guard target.defaultBuildParameters.debuggingStrategy == .modulewrap, !target.prepareForIndexing else { return }
```

Comment:
I think this should be done inside of `debuggingStrategy` getter - check whether target is for indexing and produce `nil`.

Reply:
Moved the check into the getter.